### DISCUSSION TITLE
Skeleton component

### DISCUSSION THREAD

**Discussion by alexrapro** [tamagui/tamagui#727] [Ideas]
Hi developers :)
Add a "Skeleton" component, please. This is a very frequently used component

**Comment by cerpinn** [comment]
I think there's already some work done for Skeleton here: https://github.com/tamagui/tamagui/pull/79/commits/670c4103ca425a53484b8144334db65bf2f58015
@natew can we have a mini-spec/definition-of-ready for Skeleton? I'd be happy to work on it after Toast.

**Comment by gunslingerOP** [comment]
I think this will make Tamagui a bit more competitive as it seems very bare-bones for now

**Comment by kazuma0129** [comment]
It seems to be commented out now, any updates?
https://github.com/tamagui/tamagui/blob/master/packages/tamagui/src/views/Skeleton.tsx#L9

**Comment by tomekki92** [comment]
Any updates on this?
